10_SL1/errors.yaml

```diff
@@ -36,9 +36,10 @@ Errors:
   id: "TOWER_MOVE_FAILED"
   approved: true
 
+# Deprecated, use FAN_FAILED_ID
 - code: "10106"
   title: "FAN FAILURE"
-  text: "Incorrect RPM reading of the %(failed_fans_text)s fan. Please check its wiring and connection."
```

Comment:
Vim, ze jsem to inicioval ja, ale ted jsem se zamyslel co to udela s 1.7. Martin myslim uz preklady posilal, takze kdyz to ted upravime a do 1.7 se dostanou bumpnute errory z masteru (coz se automaticky stane), tak tenhle error bude neprelozeny. 

Premyslim jestli je horsi mit tam slovo navic, nebo mit text v EN.

Reply:
Ok, commit vyhozen